src/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkSharePanel.tsx

```diff
@@ -0,0 +1,236 @@
+        </Sans>
+      </Flex>
+    )
+  }
+
+  render() {
+    const {
+      artwork: { href, imageDescription, images },
+    } = this.props
+
+    const shareImageUrl = images && images[0].url
+    const url = sd.APP_URL + href
+
+    return (
+      <Container>
+        <Box position="absolute" top={space(1)} right={space(1)}>
+          <CloseIcon name="close" onClick={this.props.onClose} />
+        </Box>
+        <Flex flexDirection="column" p={2}>
+          <Flex flexDirection="row" mb={2}>
+            <Sans size="3" weight="medium" color="black100">
+              Share
+            </Sans>
+          </Flex>
+          <Flex flexDirection="row" mb={1}>
+            <SansGrow size="2" color="black60" mr={4}>
+              <URLInput
+                type="text"
+                readOnly
+                value={url}
+                innerRef={input => (this.input = input)}
+                onClick={this.handleCopy}
+              />
+            </SansGrow>
+            <Sans size="2" weight="medium" color="black60">
+              <a onClick={this.handleCopy}>{this.state.copyLabelText}</a>
+            </Sans>
+          </Flex>
+          <Separator />
+          <Flex flexDirection="row" flexWrap="wrap">
+            {this.renderShareButton({
+              service: "facebook",
+              label: "Facebook",
+              message: "Post to Facebook",
+              url: `https://www.facebook.com/sharer/sharer.php?u=${url}`,
+            })}
+            {this.renderShareButton({
+              service: "twitter",
+              label: "Twitter",
+              message: "Share on Twitter",
+              url: `https://twitter.com/intent/tweet?original_referer=${url}&text=${imageDescription}&url=${url}&via=artsy`,
+            })}
+            {this.renderShareButton({
+              service: "mail",
+              label: "Mail",
+              message: "Share via email",
+              url: `mailto:?subject=${imageDescription}&body=Check out ${imageDescription} on Artsy: ${url}`,
+            })}
+            {this.renderShareButton({
+              service: "pinterest",
+              label: "Pinterest",
+              message: "Pin It on Pinterest",
+              url: `https://pinterest.com/pin/create/button/?url=${url}&media=${shareImageUrl}&description=${imageDescription}`,
+            })}
+            {this.renderShareButton({
+              service: "tumblr",
+              label: "Tumblr",
+              message: "",
+              url: `https://www.tumblr.com/share/photo?source=${shareImageUrl}&caption=${imageDescription}&clickthru=${url}`,
+            })}
+          </Flex>
+        </Flex>
+      </Container>
+    )
+  }
+}
+
+export const ArtworkSharePanelFragmentContainer = createFragmentContainer(
+  ArtworkSharePanel,
+  graphql`
+    fragment ArtworkSharePanel_artwork on Artwork {
+      href
+      images {
+        url
+      }
+      imageDescription: description
+      title
+      date
+      artists {
```

Comment:
I didn't notice these fields being used in this component anywhere.

Reply:
Ahh good eye -- i just realized i need to wire up the proper description using the helper laid out in force

Reply:
Actually, can remove these and go with `meta { description }` instead

Reply:
The meta tag one in MP is a tad more verbose than the specific helper in Force for the shareable component.

I think it's minor, but if the length is an issue for something shareable specifically, can always implement that in MP (such as `meta { description(format: "short") }` kinda thing, where the 'short' version doesn't include medium and dimensions, kinda thing.

Reply:
@mzikherman There’s actually a character cap (no `format`):
```
type ArtworkMeta {
  description(limit: Int = 155): String
  image: String
  title: String
}
```

155 characters seems ok, yah?